README.md

```diff
@@ -0,0 +1,98 @@
+# Ada Pets Node
+
+<!-- Note: this was based on Ada Pets React no the other way around. -->
+
+## At a Glance
+
+- Individual, [stage 1](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md#stage-1) project.
+- To be completed in class on **DATE**. No pull request is required.
+
+## Learning Goals
+
+* Reading and understanding existing JavaScript code
+* Using axios to read data from an API in JavaScript
+* Parsing through an API response in JavaScript and extracting desired information
+* Using axios to write data to an API in JavaScript
+
+## Introduction
+
+Ada Lovelace is starting a pet rescue service advertising pets in need of rescue via an API and is building a CLI app.   This app should load the list of pets automatically and present a menu with options.   The user should be able to list all pets, select a pet and see details, add new pets and remove a pet from the App once adopted.
+
+She's managed to figure out the CLI parts on her own but needs your help with the API requests.
+
+You will be using the [Pets API](https://github.com/AdaGold/pets-api) for this.
+
+### Setup
+
+1. Fork and clone this repository
+2. Run `npm install` to install dependencies
+3. Run the app with `node src/main.js`
+
+### Layout
+
+You will be adding your code to `src/adaPets.js`.  All of the functions you need to implement are currently there, just with empty bodies.
+
+There are two other files that we've provided that you won't need to change `src/main.js` and `src/result.js`.  They are worth taking a quick look at but don't worry too much if you don't understand them fully.
+* `src/main.js` is the actual driver code for the CLI app.  If you want to manually test your code you can use this.
+* `src/result.js` is the library we've written to handle results and errors for this app.  It provides three functions, two of which you will use and one of which you're going to see in the tests.  **Every helper function should make sure to call `setResult` or `setError` before each `return`.**
+  * `setResult` call this function to set the result of your function.  We use this because Axios is asynchronous.
```

Comment:
Becca and I were talking, and Becca in Wave 1, Becca was a little confused about what we should put in `setResult`. Could we put some polish by stating up here "What we pass into `setResult` and `setError` will be detailed per wave"? or something like that?